source/enum_string_mapping.h

```diff
@@ -17,9 +17,8 @@
 
 #include <string>
 
-#include "spirv/1.1/spirv.h"
-
 #include "extensions.h"
+#include "latest_version_spirv_header.h"
```

Comment:
This header should be included before other local headers. Please also add blank line, so that clang-format doesn't reorder.

Reply:
I disagree. It's a local header.
From the style guide:

> Use standard order for readability and to avoid hidden dependencies: Related header, C library, C++ library, other libraries' .h, your project's .h.